apps/ejabberd/src/mod_shared_roster_ldap.erl

```diff
@@ -179,13 +179,25 @@ get_jid_info({Subscription, Groups}, User, Server, JID) ->
         error -> {Subscription, Groups}
     end.
 
-in_subscription(Acc, User, Server, JID, Type, _Reason) ->
-    process_subscription(in, User, Server, JID, Type, Acc).
+in_subscription(Acc, _, User, Server, JID, Type, _Reason) ->
```

Comment:
https://github.com/esl/MongooseIM/blob/cac8bb99b90020bf41579817100638b04c74a83a/apps/ejabberd/src/ejabberd_sm.erl#L608

It's already a fold hook with 5 args plus Acc. Why `/7` here?